apps/remote_control/test/lexical/remote_control/code_intelligence/definition_test.exs

```diff
@@ -310,12 +358,27 @@ defmodule Lexical.RemoteControl.CodeIntelligence.DefinitionTest do
     end
   end
 
-  defp definition(project, code) do
+  defp definition(project, code, referenced_uri \\ nil) do
     with {position, code} <- pop_cursor(code),
          {:ok, document} <- subject_module(project, code),
+         :ok <- index(document, referenced_uri),
```

Comment:
this is super confusing. Sometimes you're indexing the subject, other times you're indexing the referenced file, which doesn't change. Shouldn't you _always_ index the referenced file and the subject?  I think that would make things a lot clearer and easier.